src/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ add_library(LLVMpta SHARED
 	llvm/analysis/PointsTo/Globals.cpp
 )
 
-target_link_libraries(LLVMpta PTA)
+target_link_libraries(LLVMpta PTA ${llvm_libs})
```

Comment:
Although this change makes sense, I'm getting this error when running LLVMInstrumentation (that links with `libLLVMpta.so`:

```
CommandLine Error: Option 'debug' registered more than once!
LLVM ERROR: inconsistency in registered CommandLine options
```

I suppose that every program that wants to use `libLLVMpta.so` links with LLVM, so is this a good step?